src/libigc/gnss_fix.py

```
@@ -135,17 +161,20 @@ def set_flight(self, flight: Flight):
         elif self.flight.alt_source == AltitudeSource.GNSS:
             self.alt = self.gnss_alt
         else:
-            assert(False)
+            assert False
```

Ruff (0.15.21)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)